packages/react-ui-components/src/Icon/resourceIcon.tsx

```diff
@@ -13,6 +13,11 @@ export interface ResourceIconProps extends Omit<IconProps, 'theme'> {
     readonly theme?: ResourceIconTheme;
 }
 
+/**
+ * @deprecated please refrain from using resource paths
```

Comment:
The `Icon` component is still using the ResourceIcon when defining custom icons in yaml. What is the intended replacement?